be/src/vec/common/schema_util.cpp

```diff
@@ -389,7 +389,7 @@ void inherit_column_attributes(const TabletColumn& source, TabletColumn& target,
         // add index meta
         TabletIndex index_info = *source_index_meta;
         index_info.set_escaped_escaped_index_suffix_path(target.path_info_ptr()->get_path());
-        const auto* target_index_meta = target_schema->get_inverted_index(target);
+        const auto* target_index_meta = target_schema->get_inverted_index(target, false);
```

Comment:
why set to false, and default true, add comment to explain?

Reply:
True: check columns from the variant whether supports inverted index
False: No need to check, just inherit directly

Reply:
Why not always check it?

Reply:
> Why not always check it?

1. When the variant column is created, it has a TabletIndex. The extracted column from the variant will inherit this TabletIndex.
2. The variant column itself is JSONB and does not support index,  no need to check.